frontend_v2/src/app/components/challenge/challengesubmit/challengesubmit.component.scss

```diff
@@ -122,3 +122,7 @@ th {
 #submission_option_style {
   margin-left: 1em;
 }
+
+.submission_visibility {
+  margin-left: 1em;
```

Comment:
@Alabhya268 let's not add a new css class for every change. Please put this change in existing css class or use one of the existing css class for this field.